src/main/groovy/com/contrastsecurity/VerifyContrast.groovy

```diff
@@ -114,7 +107,7 @@ class VerifyContrast extends DefaultTask {
         }
 
         for (Application application : applications.applications) {
-            if (applicationName.equals(application.name)) {
+            if (applicationName == application.name) {
```

Comment:
I thought we had to use .equals here.

Reply:
Seems like in groovy the preferred way is to use "==" http://grails.asia/groovy-compare-string

Reply:
Oh joy. Nice job, Groovy.